dvc/utils/compat.py

```diff
@@ -107,6 +112,8 @@ def _makedirs(name, mode=0o777, exist_ok=False):
     integer_types = (int, long)  # noqa: F821
     input = raw_input  # noqa: F821
     cast_bytes_py2 = cast_bytes
+    str_to_bytes = no_code
```

Comment:
why are they `no_code`? are they used anywhere at all?

Reply:
It is defunc only for py2, and is == `encode` in py3 down below.

Reply:
I mean str_to_bytes and bytes_to_str - are they used anywhere?

Reply:
Ah, good point. This is a leftover. Will remove.

Reply:
Then, please, check if we need encode/decode at all :)

Reply:
Don't need `decode`, but need `encode` for `cast_bytes`.